Makefile

```diff
@@ -165,11 +165,17 @@ O_FILES       := $(foreach f,$(S_FILES:.s=.o),build/$f) \
                  $(foreach f,$(C_FILES:.c=.o),build/$f) \
                  $(foreach f,$(wildcard baserom/*),build/$f.o)
 
-OVL_RELOC_FILES := $(shell $(CPP) $(CPPFLAGS) $(SPEC) | grep -o '[^"]*_reloc.o' )
+SEGMENTS            := $(shell $(CPP) $(CPPFLAGS) $(SPEC) | grep -o '^[ \t]*name[ \t]\+".\+"' | sed 's/.*"\(.*\)".*/\1/g' )
+SEGMENTS_OVL        := $(filter ovl_%,$(SEGMENTS))
```

Comment:
I don't really like resorting to the name either, but left it at this since as you pointed out there was already name dependence. In an earlier commit I was toying with the idea of using `flags OVL` as a means to distinguish overlays, but I dropped this for two reasons. First, to implement this cleanly it would need a tool that can receive the spec through a pipe (see prior comment on why this is apparently an issue), and second it would be a change to the spec format itself as this flag doesn't exist.

Reply:
Is changing the spec format itself an issue? (besides being more work than may be worth)
We already expanded it in the past afaik (at the very least `include_data_with_rodata` was surely not part of the "original", whatever the original is, and probably other directives)